### PR TITLE
Adding Makefile

### LINKED ISSUE #52: Figure(s)/table(s) that appear in your report should be generated programmatically for maximum reproducibility

### LINKED ISSUE #53: Move your main function up in your file. Only execution in the bottom

### LINKED ISSUE #54: Maybe put your figure into more columns instead of one single column

### LINKED ISSUE #57: Automate pipeline using Make
Create a master Make driver script that calls all other scripts in the appropriate sequence

Required for Milestone 3

Requirements:

- [ ] Place in the project's root directory and be named Makefile
- [ ]  Document well using the project README and comments inside the Makefile to explain what it does and how to use it.
- [ ] Include `all:` target so that you can easily run the entire analysis from top to bottom by running make all at the command line
- [ ] Include `clean:` target so that you can easily "undo" your analysis by running make clean at the command line

### LINKED ISSUE #58: Update README

### PR DESCRIPTION
**Pull request by zhijingjing1**

### CODE REVIEW

**Comment by yaz-saleh** [comment]
Closes  #52 #53  #54 #57 #58

File: reports/report.md [hunk 1]
```diff
@@ -99,7 +96,7 @@ High level information about the dataset variables can be found below:
 | 13    | Browser                  | Internet browser used by the visitor                                                                                                                           |
 | 14    | Region                   | Demographic region when the user accessing the site                                                                                                            |
 | 15    | TrafficType              | From which source the visitor arrived at the site                                                                                                              |
-| 16    | VisitorType              | Visitor type as “New Visitor”, “Returning Visitor”, and “Other”                                                                                                |
+| 16    | VisitorType              | Visitor type as “New Visitor,” “Returning Visitor,” and “Other”                                                                                                |
```
Comment (lephanthuymai) [inline reports/report.md:99]: Just a very minor thing: comma(,) should be placed after "

File: reports/report.md [hunk 2]
```diff
@@ -111,8 +108,8 @@ of feature 7-9: come from metrics measured by “Google Analytics” for
 each page on the site.
 
 Among those features, there are some features that might be important to
-predict the target: ‘ProductRelated\_Duration’, ‘BounceRates’,
-‘ExitRates’, ‘PageValues’, ‘Month’, ‘TrafficType’ and ‘VisitorType’.
+predict the target: ‘ProductRelated\_Duration,’ ‘BounceRates,’
+‘ExitRates,’ ‘PageValues,’ ‘Month,’ ‘TrafficType’ and ‘VisitorType.’
```
Comment (lephanthuymai) [inline reports/report.md:112]: Same as above.

File: README.md [hunk 3]
```diff
@@ -25,7 +23,7 @@ Therefore, we recommend improving this model prior to deployment in the
 real-world.
 
 The data set used in this project is the “Online Shoppers Purchasing
-Intention” dataset provided by the [Gözalan
+Intention” dataset provided by the [GÃ¶zalan
```
Comment (lephanthuymai) [inline README.md:26]: invalid unicode

**Comment by lephanthuymai** [comment]
Thanks for your hard work, @yaz-saleh and @zhijingjing1 , all changes look good, I have noticed a few minor issues, you can just leave them there.

File: reports/report.md (hunk 1 above)
Comment (yaz-saleh) [reply reports/report.md:99]: I agree.. this was auto formatted when I save unfortunately. I will look into why it is happening (probably the linter or formatter)

File: README.md (hunk 3 above)
Comment (yaz-saleh) [reply README.md:26]: Hmm I swear I had fixed this last week. It seems to reset (again probably the formatter). I will fix it

**Pull request merged**